### PR TITLE
Fix managed certs validation prompt

### PR DESCRIPTION
**Pull request by lil131**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

containerapp

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by howang-ms — APPROVED** [review]

File: src/containerapp/azext_containerapp/custom.py [hunk 1]
```diff
@@ -2930,9 +2930,9 @@ def create_managed_certificate(cmd, name, resource_group_name, hostname, validat
         cert_name = generate_randomized_managed_cert_name(hostname, resource_group_name)
         if not check_managed_cert_name_availability(cmd, resource_group_name, name, certificate_name):
             cert_name = None
-    certificate_envelop = prepare_managed_certificate_envelop(cmd, name, resource_group_name, hostname, validation_method)
+    certificate_envelop = prepare_managed_certificate_envelop(cmd, name, resource_group_name, hostname, validation_method.upper())
```
Comment (zhoxing-ms) [inline src/containerapp/azext_containerapp/custom.py:2933]: I'd like to confirm if there is a possibility for `validation_method` being `None`?

**Comment by zhoxing-ms** [comment]
If you want to release a new version for this extension, please also upgrade the version defined in `setup.py`

**Comment by zhoxing-ms** [comment]
> We decided not to bump the version in this PR. Should I mark the notes of changes in History.rst as "Upcoming"?

Yes, sounds good!

**Comment by lil131** [comment]
> > We decided not to bump the version in this PR. Should I mark the notes of changes in History.rst as "Upcoming"?
> 
> Yes, sounds good!

Marked as Upcoming. Could you please take another look when you get a chance? Thanks!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**